algo/src/main/java/com/hartwig/actin/algo/evaluation/tumor/TumorMetastasisEvaluator.java

```diff
@@ -7,9 +7,10 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-public class TumorUtil {
+public final class TumorMetastasisEvaluator {
```

Comment:
I looked into making this a function that takes in a `MetastasesEvaluationFunction`, but every step in that direction seemed to reintroduce the duplication that I was trying to clean up. Instead, I went with @kduyvesteyn's suggested rename, which should improve readability.

Reply:
I can see your point. I've thought of alternatives but can't think of a good one either. You could make the function non-static and create an instance every time the parent evaluation function is called but that would be worse even. And the function you end up does not resemble a normal evaluation function anymore either. Fine with this rename